client/js/verification-portal/get-resource-to-verify-button.js

```diff
@@ -87,8 +87,10 @@ const populateForm = ({ changeFieldValue, resource }) => {
 		_.get(verifiedData, 'primaryContactPerson.verified'),
 	)
 
-	_.forEach(services, (value, key) => {
-		changeFieldValue(`services.${key}`, 1)
+	_.forEach(services, value => {
+		resource.services[value.id]
+			? changeFieldValue(`services.${value.id}`, 1)
+			: changeFieldValue(`services.${value.id}`, 0)
```

Comment:
Let's change the `1` to `true` and the `0` to `false` to match what is returned from the server. i.e. the values in the response should match the values in the redux form.

Values from response:
<img width="278" alt="screen shot 2018-06-17 at 11 41 37 am" src="https://user-images.githubusercontent.com/6378435/41511029-31882050-7224-11e8-9e36-652077fd6ad1.png">

Values in redux form:
<img width="261" alt="screen shot 2018-06-17 at 11 41 44 am" src="https://user-images.githubusercontent.com/6378435/41511033-3c687f38-7224-11e8-9d27-673c60887116.png">

This is what happens when you update the form:
<img width="261" alt="screen shot 2018-06-17 at 11 41 44 am" src="https://user-images.githubusercontent.com/6378435/41511037-447ef990-7224-11e8-9e5a-c7fdae3dbe0d.png">